tests/unittests/GraphOptzTest.cpp

```diff
@@ -184,6 +184,34 @@ TEST_F(GraphOptz, liveCodeNotEliminated) {
   EXPECT_EQ(mod_.getPlaceholders().size(), 3);
 }
 
+TEST_F(GraphOptz, optimizeBatchNormAfterConvAndReshape) {
```

Comment:
Test LG, can we add one for NCHW, and maybe also one where we think the opt shouldn't kick in? I think the conditions you have right now aren't right, and so it would actually cause the opt to kick in when it shouldn't and should fail a test. Like make the batch, channel, H all the same value to make sure we're comparing the right indexing.

Reply:
I have added 3 tests: two positive tests and one negative test.